### PR TITLE
Implement simple maps support

### PR DESCRIPTION
**Pull request by Fabi2607**
The location data provided by the database can be used to create an
Map intent and show the summit in a user defined mapping
application (e.g. OsmAnd, Google Maps).

Signed-off-by: Fabian Kantereit <fabian.kantereit@tsaro.io>

### CODE REVIEW

File: app/src/main/java/com/example/paetz/yacguide/RouteActivity.java [hunk 1]
```diff
@@ -50,6 +51,15 @@ public void back(View v) {
         finish();
     }
 
+    public void showMap(View v) {
+        Uri gmmIntentUri = Uri.parse("geo:"+ _rock.getLatitude() + "," + _rock.getLongitude());
```
Comment (paetz) [inline app/src/main/java/com/example/paetz/yacguide/RouteActivity.java]: could be final

File: app/src/main/java/com/example/paetz/yacguide/RouteActivity.java [hunk 2]
```diff
@@ -50,6 +51,15 @@ public void back(View v) {
...
+        Intent mapIntent = new Intent(Intent.ACTION_VIEW, gmmIntentUri);
+        if (mapIntent.resolveActivity(getPackageManager()) != null) {
+            startActivity(mapIntent);
+        }
```
Comment (paetz) [inline app/src/main/java/com/example/paetz/yacguide/RouteActivity.java]: If there is no map tool available, you should show an according error message (using Toast) like "Keine Karten-App verfügbar."

File: app/src/main/java/com/example/paetz/yacguide/RouteActivity.java [hunk 3]
```diff
@@ -50,6 +51,15 @@ public void back(View v) {
         finish();
     }
 
+    public void showMap(View v) {
+        Uri gmmIntentUri = Uri.parse("geo:"+ _rock.getLatitude() + "," + _rock.getLongitude());
+        Intent mapIntent = new Intent(Intent.ACTION_VIEW, gmmIntentUri);
+        if (mapIntent.resolveActivity(getPackageManager()) != null) {
+            startActivity(mapIntent);
```
Comment (paetz) [inline app/src/main/java/com/example/paetz/yacguide/RouteActivity.java]: I tested with google maps on the emulator and the coords were NOT represented with a marker. Can we somehow make sure that the map tool provides a proper marker for the given coords?

Comment (Fabi2607) [reply app/src/main/java/com/example/paetz/yacguide/RouteActivity.java]: Good question, we could request a whitelisted set of map tools...

**Comment by Fabi2607** [comment]
I will address your comments, as soon as I find time. Just wanted to share my modifications in case someone finds them useful.

File: app/src/main/java/com/example/paetz/yacguide/RouteActivity.java (hunk 3 above)
Comment (Fabi2607) [reply app/src/main/java/com/example/paetz/yacguide/RouteActivity.java]: I investigated a bit more... There is absolutely now way to tell the Uri that there should be a marker. We can either query different apps one by one, leave it as is or integrate a map toolkit (which I considered too much work right now)

Comment (paetz) [reply app/src/main/java/com/example/paetz/yacguide/RouteActivity.java]: Ok, then we leave it as is for now, since it's better than nothing

**Pull request merged**